src/main/java/forestry/core/tiles/TileAnalyzer.java

```diff
@@ -54,7 +55,7 @@
 public class TileAnalyzer extends TilePowered implements ISidedInventory, ILiquidTankTile, IItemStackDisplay {
 	private static final int TIME_TO_ANALYZE = 125;
 	private static final int HONEY_REQUIRED = 100;
-	private static final int ENERGY_PER_WORK_CYCLE = 20320;
+	private static final int ENERGY_PER_WORK_CYCLE = Config.analyzerEnergyPerWork;
```

Comment:
Please replace every appearance of `ENERGY_PER_WORK_CYCLE` with `Config.analyzerEnergyPerWork`. And remove `ENERGY_PER_WORK_CYCLE`.